react.js

```diff
@@ -1,42 +1,46 @@
 const config = require('./config');
-const base = require('./index');
 
 const ERROR = 'error';
 const OFF = 'off';
 
-module.exports = Object.assign({}, base, {
-  plugins: base.plugins.concat(
+module.exports = {
+  plugins: [
     'css-modules',
     'jsx-a11y',
     'react',
-    'react-hooks'
-  ),
+    'react-hooks',
+    'sort-destructure-keys'
```

Comment:
This was the last missing piece so that everything related to React props is sorted (JSX, prop types, prop destructuring). `sort-keys` from ESLint is a bit too aggressive IMO.